docs/contributing-analyzer.md

```diff
@@ -24,19 +24,13 @@ Visual Studio 2022 version 17.6+ is required to build the project (due to source
     - .NET Framework 4.8 SDK
     - .NET Framework 4.8 Targeting pack
     - .NET Framework 4.7.2 Targeting pack
```

Comment:
Why do we need this? 
Our UTs are running against 4.8, right?
Is it a specific project in the ITs that targets 4.7.2?